src/domain/entities/noteList.ts

```diff
@@ -0,0 +1,5 @@
+import type { Note } from '@domain/entities/note.js';
+
+export type NoteList = {
```

Comment:
Add docs pls

Reply:
Done

Reply:
Description repeats variable name. Describe what does "Note List" means in real world.

Reply:
Improved description